store/src/java/com/zimbra/cs/mailbox/CalendarItem.java

```diff
@@ -3470,6 +3470,24 @@ private void updateLocalExceptionsWhichMatchSeriesReply(Invite reply) throws Ser
         }
     }
 
+    private Instance getNearestInstance(Collection<Instance> instancesNear,
+        Instance replyInstance) {
+        long min = replyInstance.getStart();
```

Comment:
I'd probably make this something like a year long, with a comment that we assume everything in instancesNear will be much closer than that.

Reply:
Done